crates/nu-command/src/conversions/into/cellpath.rs

```diff
@@ -0,0 +1,125 @@
+use nu_protocol::ast::{CellPath, PathMember};
```

Comment:
I am not sure if this file is bycatch from another PR? (not sure if we landed or reverted something there)

Reply:
Yeah this is because `into cellpath` was reverted: https://github.com/nushell/nushell/pull/7523